packages/testing/config/jest/web/RedwoodWebJestEnv.js

```diff
@@ -0,0 +1,19 @@
+const Environment = require('jest-environment-jsdom')
+
+/**
+ *
+ * This just adds TextEncoder and TextDecoder to the jsdom environment
+ * to prevent errors when running tests parallely
+ *
+ */
+module.exports = class CustomTestEnvironment extends Environment {
+  async setup() {
+    await super.setup()
+
+    if (typeof this.global.TextEncoder === 'undefined') {
+      const { TextEncoder, TextDecoder } = require('util')
+      this.global.TextEncoder = TextEncoder
+      this.global.TextDecoder = TextDecoder
+    }
+  }
+}
```

Comment:
I know you were working on solving a larger problem at first, and then scaled back the code here. But now this seems like overkill. I'd like to remove this file and revert to handling the TextDecoder in web/index.js:
https://github.com/redwoodjs/redwood/blob/56829914fa681e4d0f4fe7c6c69f48b79e8186b9/packages/testing/config/jest/web/index.js#L19

Reply:
From my tests DP this is the more consistent and reliable way of setting globals that jest would use internally. I think we should preserve setting up the environment like this, atleast until jest/jsdom have a proper solution. 

+ consistency with how it's configured on the api side.

Reply:
Dom and I both feel like the TextDecoder issue is actually a bug introduced by Jest recently — we shouldn't have to include it at all. Next upgrade cycle I was going to remove and see if it was fixed.

I'd like to revert and use default 'jest-environment-jsdom'. I just don't see a strong reason at this time to override the Jest env. And it's late in the roadmap to be making config changes imho.

Reply:
✅